index.bs

```diff
@@ -4427,7 +4427,13 @@ RogueList when performing ECDAA-Verify (see section 3.6 in [[!FIDOEcdaaAlgorithm
 [[FIDOMetadataService]] provides one way to access such information.
 
 
+## credentialId Unsigned ## {#credentialIdSecurity}
 
+The credential ID is not signed.
+This is not a problem because all that would happen if an authticator returns
+the wrong crednetial ID is that the RP will not look up the correct
```

Comment:
Typo: crednetial
Suggest: RP -> [=[RP] =]

Reply:
I think a more interesting case to point out is if a MitM were to manipulate the credential ID (the conclusion is the same), so I would probably write this as something like "[...] if an authenticator returns the wrong credential ID, or if an attacker intercepts and manipulates the credential ID, is that the RP [...]"